dev/deps/spark-deps-hadoop-2.7

```diff
@@ -26,8 +26,7 @@ breeze-macros_2.12-0.13.2.jar
 breeze_2.12-0.13.2.jar
 chill-java-0.9.3.jar
 chill_2.12-0.9.3.jar
-commons-beanutils-1.7.0.jar
-commons-beanutils-core-1.8.0.jar
```

Comment:
Could you remove the following line too?
- https://github.com/apache/spark/blob/master/LICENSE-binary#L305